front/src/common/redux/services/authenticationService.js

```diff
@@ -0,0 +1,19 @@
+// import axios from 'axios';
+export const loginUserService = (request) => {
+    const LOGIN_API_ENDPOINT = 'http://127.0.0.1:5000/auth/login';
+    const parameters = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(request.user)
+    };
+
+    return fetch(LOGIN_API_ENDPOINT, parameters)
```

Comment:
Please, use `axios` library for making HTTP requests. Also, create a separate file where you configure default `axios` instance and don't hardcode URL of the backend in the string because url can change. Instead take it from the environment variable